datepicker/languages.js

```diff
@@ -97,5 +97,14 @@ export default {
       'date': 'Izberite datum',
       'dateRange': 'Izberite razpon med 2 datumoma'
     }
+  },
+  'pl': {
+    'days': ['Niedz', 'Pon', 'Wt', 'Śr', 'Czw', 'Pt', 'Sob'],
+    'months': ['Sty', 'Lut', 'Mar', 'Kwi', 'Maj', 'Cze', 'Lip', 'Sie', 'Wrz', 'Paź', 'Lis', 'Gru'],
+    'pickers': ['następne 7 dni', 'następne 30 dni', 'wcześniejsze 7 dni', 'wcześniejsze 30 dni'],
```

Comment:
Maybe 'poprzednie 7 dni' etc. would be more appropriate ?

Reply:
I suggest shorten day of the week names and use of two-letter calendar symbols

```
  'pl': {
    'days': ['Nd', 'Pn', 'Wt', 'Śr', 'Cz', 'Pt', 'So'],
    'months': ['Sty', 'Lut', 'Mar', 'Kwi', 'Maj', 'Cze', 'Lip', 'Sie', 'Wrz', 'Paź', 'Lis', 'Gru'],
    'pickers': ['następne 7 dni', 'następne 30 dni', 'poprzednie 7 dni', 'poprzednie 30 dni'],
```